packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -55,3 +60,32 @@ export class NodeTracerProvider extends BasicTracerProvider {
     super.register(config);
   }
 }
+
+/**
+ * Two layer merge.
+ * First, for user supplied config of plugin(s) that are loaded by default,
+ * merge the user supplied and default configs of said plugin(s).
+ * Then merge the results with the default plugins.
+ * @returns 2-layer deep merge of default and user supplied plugins.
+ */
+export function mergePlugins(
+  defaultPlugins: Plugins,
+  userSuppliedPlugins: Plugins
+): Plugins {
+  let mergedUserSuppliedPlugins: Plugins = {};
```

Comment:
`s/let/const`?

Reply:
Thanks, isn't `gts` supposed to catch this sort of thing? 🤔